lib/browser/api/auto-updater/squirrel-update-win.ts

```diff
@@ -94,7 +92,6 @@ export function checkForUpdate (updateURL: string, callback: (error: Error | nul
       update = (ref = JSON.parse(json!)) != null ? (ref1 = ref.releasesToApply) != null ? typeof ref1.pop === 'function' ? ref1.pop() : undefined : undefined : undefined;
     } catch {
       // Disabled for backwards compatibility:
```

Comment:
Same thing in this file, these comment lines preceding the eslint comment line can be removed.